README.md

```diff
@@ -1 +1,187 @@
-# Hello-World
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```

Comment:
- [ ] Great work using the README template provided by Microverse :clap: However it would be nice to remove this commented-out description. It just describes how to write your README file in a professional manner. Therefore remove them to make your README file looks more professional.

Reply:
Removed comments